mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -172,3 +172,54 @@ def accept_or_reject(self):
             logger.exception(f'Check {self.object_id} could not be actioned')
             self.add_error(None, _('There was an error with your request.'))
             return False
+
+
+class AssignCheckToUserForm(GARequestErrorReportingMixin, forms.Form):
+    assignment = forms.CharField(max_length=10)
```

Comment:
you could set choices on this field to automatically disallow anything but 'assign' and 'unassign'. automatic validation is easier to deal with and shows you the options up-front

Reply:
Cool - updated to use `choiceField` and got rid of the else